daemon/daemon.go

```diff
@@ -916,8 +915,14 @@ func NewDaemon(ctx context.Context, config *config.Config, pluginStore *plugin.S
 		Backoff: backoffConfig,
 	}
 	gopts := []grpc.DialOption{
-		// WithBlock makes sure that the following containerd request
-		// is reliable.
+		// ------------------------------------------------------------------
+		// options below are copied from containerd client's default options
+		//
+		// TODO(thaJeztah): update this list once https://github.com/containerd/containerd/pull/10250/commits/63b46881753588624b2eac986660458318581330 is in the 1.7 release.
```

Comment:
Oh! Maybe I can remove  more now, because we switched to the containerd v2 module; let me check before we merge.